apps/studio/components/layouts/AppLayout/RestrictionBanner.tsx

```diff
@@ -27,7 +27,7 @@ export const RestrictionBanner = () => {
         {currentOrg.restriction_status === 'restricted' ? <CriticalIcon /> : <WarningIcon />}
         <span>
           {currentOrg.restriction_status === 'grace_period' &&
-            `Your organization has exceeded its quota. You are given a grace period until ${dayjs(currentOrg.restriction_data['grace_period_end']).format('DD MMM, YYYY')}`}
+            `Your organization has exceeded its quota. You are given a grace period until ${dayjs(currentOrg.restriction_data!['grace_period_end']).format('DD MMM, YYYY')}`}
```

Comment:
Was always nullable, but if `restriction_status` is null, `restriction_data` should not be there either.